### PR TITLE
JavaPoet.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,83 +52,83 @@ Version 2.3.0 *(2013-11-24)*
...
  * Fix: `stringLiteral` now correctly handles escapes and control characters.
 
 
-Version 2.1.0 *(2013-07-15)*
+JavaWriter 2.1.0 *(2013-07-15)*
 ----------------------------
 
  * New: All methods now take a `Set` of `Modifier`s rather than an `int`. The `int` methods are
-   now deprecated for removal in version 3.0.
+   now deprecated for removal in JavaPoet 1.0.
```
Comment (JakeWharton) [inline CHANGELOG.md]: Revert

File: pom.xml [hunk 2]
```diff
@@ -10,12 +10,12 @@
   </parent>
 
   <groupId>com.squareup</groupId>
-  <artifactId>javawriter</artifactId>
+  <artifactId>javapoet</artifactId>
   <version>3.0.0-SNAPSHOT</version>
```
Comment (JakeWharton) [inline pom.xml]: Needs updated to 1.0.0-SNAPSHOT

**Comment by JakeWharton** [comment]
LGTM

File: README.md [hunk 3]
```diff
@@ -13,30 +13,40 @@ Example
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Raven {
+  public void main() {
+    List<String> verses = new ArrayList<String><>();
+    verses.add("Once upon a midnight dreary, while I pondered, weak and weary...");
+    verses.add("Over many a quaint and curious volume of forgotten lore—");
+    return verses;
```
Comment (roman-mazur) [inline README.md]: Method return type is `void`. Does not compile. :bomb:

Comment (JakeWharton) [inline README.md]: https://github.com/square/javapoet/pull/176

**Pull request merged**